src/playground/Widgets/Tasks/index.tsx

```diff
@@ -83,21 +84,12 @@ export const TasksInsight = forwardRef<HTMLDivElement, TasksInsightProps>(
             />
           ))}
         </div>
-        <div className="flex flex-col gap-3">
-          {!tasks?.length ? (
-            <p>{emptyStateText}</p>
-          ) : (
-            tasks.slice(0, 5).map((task, i) => (
-              <div
-                key={`${task} ${i}`}
-                className="flex flex-row items-center gap-3"
-              >
-                <div className="border-f1-border h-5 min-w-5 rounded-md border border-solid" />
-                <p className="truncate font-medium">{task}</p>
-              </div>
-            ))
-          )}
-        </div>
+        {(inProgressTasks.length || pendingTasks.length) && (
+          <TasksList
+            inProgressTasks={inProgressTasks}
+            pendingTasks={pendingTasks}
+          />
+        )}
```

Comment:
I think we don't need an empty state text here, it's self-explanatory seeing the counters already, and we're saving some vertical space as well.

<img width="475" alt="Screenshot 2024-09-16 at 13 18 31" src="https://github.com/user-attachments/assets/ae312af4-719f-4fd1-8a77-8abea2afba08">